repository-data/application/src/main/resources/hcm-config/namespaces/hee/guidanceDocument.yaml

```diff
@@ -0,0 +1,183 @@
+definitions:
+  config:
+    /hippo:namespaces/hee/guidance:
+      jcr:primaryType: hipposysedit:templatetype
+      jcr:mixinTypes: ['editor:editable', 'mix:referenceable']
+      jcr:uuid: 75f31343-4f4b-4aa6-8e80-28067c96ae65
+      /hipposysedit:nodetype:
+        jcr:primaryType: hippo:handle
+        jcr:mixinTypes: ['mix:referenceable']
+        jcr:uuid: 4ee4631b-efde-4f48-bcfa-f85feb78d3f3
+        /hipposysedit:nodetype:
+          jcr:primaryType: hipposysedit:nodetype
+          jcr:mixinTypes: ['mix:referenceable', 'hipposysedit:remodel']
+          jcr:uuid: 4ce03344-e1b5-49c4-b10c-54c4eb6187d0
+          hipposysedit:node: true
+          hipposysedit:supertype: ['hee:basedocument', 'hippostd:relaxed', 'hippotranslation:translated']
+          hipposysedit:uri: http://www.heecmsplatform.com/hee/nt/1.0
+          /title:
+            jcr:primaryType: hipposysedit:field
+            hipposysedit:mandatory: false
+            hipposysedit:multiple: false
+            hipposysedit:ordered: false
+            hipposysedit:path: hee:title
+            hipposysedit:primary: false
+            hipposysedit:type: String
+            hipposysedit:validators: [required]
+          /summary:
+            jcr:primaryType: hipposysedit:field
+            hipposysedit:mandatory: false
+            hipposysedit:multiple: false
+            hipposysedit:ordered: false
+            hipposysedit:path: hee:summary
+            hipposysedit:primary: false
+            hipposysedit:type: Text
+            hipposysedit:validators: [required]
+          /contentBlocks:
```

Comment:
Just wondering if `pageLastNextReview` compound type can be a separate field within `guidance` document type rather than part of `contentBlocks` field as I think it's for Page level rather than at the Section level ?

Reply:
@smanivasag This case is a bit of ambiguous because I saw some design when designer put the pagelastNextReview on the top before the main html content. So if we put it into the ContentBlock that could be easier to control the order. I am not sure that make sense ?

Reply:
@thojava - I see what you mean, I could few pages having the dates at the bottom of the page (but, I am not sure these are Guidance pages) and few at the top of the page (right below Summary). But, based on the wireframes, for Guidance pages, so far I could see dates at the top of the pages (right below the Summary). Also, I think the location of these dates on the pages should be controlled by the (FTL) templates (rather than  by the content authors) in order to keep the consistency of page layout. Suggest to stick to it for now unless the designer says otherwise. Hope that makes sense ? Thanks!

Reply:
@smanivasag Yeap agree with this. I will justify this on next PR